.vale.ini

```
@@ -0,0 +1,7 @@
+StylesPath = styles
+
+MinAlertLevel = suggestion
+Packages = Microsoft, write-good
+
+[*]
```

Let's limit Vale to Markdown files to start with. We can expand to cover code files later.

```suggestion
[*.md]
```